.github/workflows/macos-windows.yml

```diff
@@ -0,0 +1,62 @@
+name: macOS and Windows
```

Comment:
It might be better to pair together macOS and Ubuntu together in the future when we do drop azure.